packages/nx-plugin/src/schematics/plugin/plugin.ts

```diff
@@ -113,18 +116,22 @@ function addFiles(options: NormalizedSchema): Rule {
           tmpl: '',
           offsetFromRoot: offsetFromRoot(options.projectRoot)
         }),
+        options.unitTestRunner === 'none'
+          ? filter(file => !file.endsWith('.spec.tsx'))
+          : noop(),
```

Comment:
The plugin schematic doesn't have any spec files in the template folder. It's just the `index.ts`. So this part shouldn't be needed. (If it is needed, it shouldn't have `.spec.tsx` 😉 )
```suggestion
```

Reply:
😅 fixed and pushed.